fe/fe-core/src/main/java/com/starrocks/server/MetadataMgr.java

```diff
@@ -438,6 +439,35 @@ public void dropTemporaryTable(DropTemporaryTableStmt stmt) {
         });
     }
 
+    public void cleanTemporaryTables(CleanTemporaryTableStmt stmt) {
+        Preconditions.checkArgument(stmt.getSessionId() != null,
+                "session id should not be null in DropTemporaryTableStmt");
+        cleanTemporaryTables(stmt.getSessionId());
+    }
+
+    public void cleanTemporaryTables(UUID sessionId) {
+        TemporaryTableMgr temporaryTableMgr = GlobalStateMgr.getCurrentState().getTemporaryTableMgr();
+        com.google.common.collect.Table<Long, String, Long> allTables = temporaryTableMgr.getTemporaryTables(sessionId);
+
+        for (Long databaseId : allTables.rowKeySet()) {
+            Database database = localMetastore.getDb(databaseId);
+            if (database == null) {
+                continue;
```

Comment:
if we continue here, there maybe some tmp table info remain in temporaryTableMgr and cannot be cleaned before restart?

Reply:
> if we continue here, there maybe some tmp table info remain in temporaryTableMgr and cannot be cleaned before restart?

Good question. If we forcefully delete the database, this branch will be reached. At this time, we need to clean up the record in TemporaryTableMgr.

Reply:
fixed